webhook_to_fedora_messaging/database.py

```diff
@@ -8,14 +8,27 @@
 Import the functions we will use in the main code and in migrations.
 """
 
+from collections.abc import Iterator
+
+from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy_helpers import (  # noqa: F401
     Base,
+    DatabaseManager,
     exists_in_db,
     get_or_create,
     is_sqlite,
     update_or_create,
 )
-from sqlalchemy_helpers.flask_ext import DatabaseExtension, first_or_404, get_or_404  # noqa: F401
+from sqlalchemy_helpers.fastapi import make_db_session
+
+
+db = None
+
+
+async def setup_database():
+    await db.sync()
 
 
-db = DatabaseExtension()
+async def get_session() -> Iterator[AsyncSession]:
```

Comment:
I'm getting a type error in my IDE here, it looks like it should be `AsyncIterator`, not `Iterator`. I'll update `sqlalchemy-helper`'s documentation accordingly.